doc/content/source/turbulence/auxkernels/WallDistanceAux.md

```diff
@@ -0,0 +1,52 @@
+# WallDistanceAux
+
+`WallDistanceAux` is an auxiliary finite volume (FV) kernel that **computes the distance from each
+quadrature point to the nearest solid wall boundary**. The resulting distance field can be used
+by near-wall turbulence models, wall functions, or any model that requires a wall-normal length
+scale (e.g., for computing $y^+$ or two-layer blending functions).
+
+More precisely, for each quadrature point located at position $\mathbf{x}_p$ inside an element,
+`WallDistanceAux` computes
+
+\begin{equation}
+d(\mathbf{x}_p) = \min_{\mathbf{x}_w \in \Gamma_{\text{walls}}} \lVert \mathbf{x}_p - \mathbf{x}_w \rVert,
+\end{equation}
+
+where $\Gamma_{\text{walls}}$ is the union of all boundary faces designated as solid walls, and
+$\mathbf{x}_w$ is taken as the centroid of a boundary face in the mesh.
+
+The computed value is stored in the auxiliary variable associated with this kernel.
+
+## Usage
+
+### Wall boundaries
+
+The set of boundaries that represent solid walls is specified with the `walls` parameter:
+these are the boundaries to which distances are computed.
+
+- `walls`: a list of boundary names that correspond to solid walls.
+
```

Comment:
Feels like this is redundant here, the exact same is claimed in the sentences above.